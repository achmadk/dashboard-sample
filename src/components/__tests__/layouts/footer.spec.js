import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import chai from 'chai'

import FooterContainer, { footerComponent } from '@/components/layouts/footer'
import FooterComponent from '@/components/layouts/footer/footer'

describe('Footer Layout Container', () => {
  it('render correctly', () => {
    const wrapper = shallow(<FooterContainer />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it('return layout component successfully', () => {
    const result = footerComponent()
    chai.expect(result).to.exist
  })
})

describe('Footer Layout Component', () => {
  it('render correctly', () => {
    const wrapper = shallow(<FooterComponent />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
