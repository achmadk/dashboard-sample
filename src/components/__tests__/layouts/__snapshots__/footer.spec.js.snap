// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Layout Component render correctly 1`] = `
<section
  className="app-footer"
>
  <div
    className="container-fluid"
  >
    <span
      className="float-left"
    >
      <span>
        Copyright Â© 
        <a
          className="brand"
          href="http://procura.id"
          rel="noopener noreferrer"
          target="_blank"
        >
          ProcurA
        </a>
         2019
      </span>
    </span>
    <span
      className="float-right"
    >
      <span>
        Built with Love 
        <i
          className="material-icons"
        >
          favorite_border
        </i>
      </span>
    </span>
  </div>
</section>
`;

exports[`Footer Layout Container render correctly 1`] = `
<React.Suspense
  fallback={
    <div>
      Loading...
    </div>
  }
>
  <React.Lazy />
</React.Suspense>
`;
